2.8.0/todo/result_cpu (x86).md

```diff
@@ -28,12 +28,14 @@ The categories below are as follows:
 ### deprecation
 ### new features
 ### improvements
+- [CPU][Brgemm] add s8s8 GEMM microkernel API ([#154358](https://github.com/pytorch/pytorch/pull/154358))
```

Comment:
do you mind removing the tags in brackets here? The release notes runbook recommends formatting titles without these

Reply:
Removed